src/main/webapp/maps.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Styled Map Types</title>
+  <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
+  <meta charset="utf-8">
+  <style>
+    /* Always set the map height explicitly to define the size of the div
+     * element that contains the map. */
+    #map {
+      height: 80%;
+      border: thin solid black;
+    }
+    /* Optional: Makes the sample page fill the window. */
+    html, body {
+      height: 80%;
+      margin: 0;
+      padding: 10px;
+    }
+  </style>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBxd60VBUhGDGcDgEEwgyFkEfVtcG6xiZU"></script>
+  <script src="/js/maps-page-loader.js"></script>
+<!--  <style>
+    #map {
+      width: 500px;
+      height: 500px;
+      border: thin solid black;
+    }
+  </style>-->
+</head>
+<body onload="createMap();">
+<h1>Hello Google Maps</h1>
```

Comment:
generally better to indent the html elements too. it's easier to see which elements are inside which elements. same for the rest of the html elements.
```suggestion
  <h1>Hello Google Maps</h1>
```